### ISSUE TITLE
Can't open transport stations window after switching save

### ISSUE THREAD

**Issue by starfi5h** [LittleSaya/DSPTransportStat#4]
### Steps to reproduce
Load a save
Exit Game
Load a save
Ctrl + F can't open tansport stat window now

Only use DSPTransportStat - v 0.0.10

**Comment by LittleSaya** [comment]
Yes, I have found this bug. I think I used a wrong approach to create the window, and it can not be recreated after being destroyed. I'm currently looking into this problem.

**Comment by LittleSaya** [comment]
I just uploaded a new version, plase update and see if this problem got fixed.
